src/java.base/share/classes/java/lang/InterruptedException.java

```diff
@@ -32,10 +32,10 @@
  * thread has been interrupted, and if so, to immediately throw
  * this exception.  The following code can be used to achieve
  * this effect:
- * <pre>
- *  if (Thread.interrupted())  // Clears interrupted status!
- *      throw new InterruptedException();
- * </pre>
+ * {@snippet:
```

Comment:
I recommend using `{@snippet lang=java :` instead, like in https://github.com/openjdk/jdk/blob/6520843f86f638fe4d1e5b3358fab5799daca654/src/java.compiler/share/classes/javax/tools/JavaFileManager.java#L84